### ISSUE TITLE
[CRASH] After opening Nether portal and passing through...

### ISSUE THREAD

**Issue by hron84** [MightyPirates/BedrockOres#37]
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  ShetiPhian-ASM (shetiphiancore-1.10.0-3.3.11.jar)
  ForgelinPlugin (Forgelin-1.5.1.jar)
  CTMCorePlugin (CTM-MC1.10.2-0.2.2.24.jar)
  LoadingPlugin (Quark-r1.1-70.jar)
  SHLoadingPlugin (ScalingHealth-1.11.2-1.0.8-52.jar)
  EnderCorePlugin (EnderCore-1.10.2-0.4.1.66-beta.jar)
  FMLPlugin (InventoryTweaks-1.61-58.jar)
  IvToolkit (IvToolkit-1.3.3-1.10.jar)
  AstralCore (astralsorcery-1.4.5.jar)
Contact their authors BEFORE contacting forge

// Surprise! Haha. Well, this is awkward.

Time: 9/10/17 9:48 AM
Description: Exception ticking world

java.lang.ArrayIndexOutOfBoundsException: Array index out of range: 16
	at gnu.trove.list.array.TFloatArrayList.get(TFloatArrayList.java:281)
	at li.cil.bedrockores.common.world.WorldGeneratorBedrockOre.generateImpl(WorldGeneratorBedrockOre.java:194)
	at li.cil.bedrockores.common.world.WorldGeneratorBedrockOre.generate(WorldGeneratorBedrockOre.java:51)
	at net.minecraftforge.fml.common.registry.GameRegistry.generateWorld(GameRegistry.java:120)
	at net.minecraft.world.chunk.Chunk.func_186034_a(Chunk.java:1001)
	at net.minecraft.world.chunk.Chunk.func_186030_a(Chunk.java:964)
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:154)
	at net.minecraft.world.World.func_72964_e(World.java:303)
	at net.minecraft.world.World.func_175726_f(World.java:298)
	at net.minecraft.world.World.func_180495_p(World.java:842)
	at net.minecraft.world.chunk.Chunk.func_76594_o(Chunk.java:1257)
	at net.minecraft.world.WorldServer.func_147456_g(WorldServer.java:393)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:220)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:698)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:149)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at gnu.trove.list.array.TFloatArrayList.get(TFloatArrayList.java:281)
	at li.cil.bedrockores.common.world.WorldGeneratorBedrockOre.generateImpl(WorldGeneratorBedrockOre.java:194)
	at li.cil.bedrockores.common.world.WorldGeneratorBedrockOre.generate(WorldGeneratorBedrockOre.java:51)
	at net.minecraftforge.fml.common.registry.GameRegistry.generateWorld(GameRegistry.java:120)
	at net.minecraft.world.chunk.Chunk.func_186034_a(Chunk.java:1001)
	at net.minecraft.world.chunk.Chunk.func_186030_a(Chunk.java:964)
	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:154)
	at net.minecraft.world.World.func_72964_e(World.java:303)
	at net.minecraft.world.World.func_175726_f(World.java:298)
	at net.minecraft.world.World.func_180495_p(World.java:842)
	at net.minecraft.world.chunk.Chunk.func_76594_o(Chunk.java:1257)
	at net.minecraft.world.WorldServer.func_147456_g(WorldServer.java:393)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:220)

-- Affected level --
Details:
	Level name: Naruto
	All players: 1 total; [EntityPlayerMP['hron84'/296, l='Naruto', x=87.50, y=40.00, z=32.15]]
	Chunk stats: ServerChunkCache: 428 Drop: 0
	Level seed: -1968799663
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (0,0,0), Chunk: (at 0,0,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 0 game time, 0 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: ~~ERROR~~ NullPointerException: null
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:698)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:149)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471)
	at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
	Minecraft Version: 1.10.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 781549552 bytes (745 MB) / 3126329344 bytes (2981 MB) up to 3126329344 bytes (2981 MB)
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx3072m -Xms256m -XX:PermSize=256m
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP 9.32 Powered by Forge 12.18.3.2422 60 mods loaded, 60 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.10.2-12.18.3.2422.jar) 
	UCHIJAAAA	Forge{12.18.3.2422} [Minecraft Forge] (forge-1.10.2-12.18.3.2422.jar) 
	UCHIJAAAA	ivtoolkit{1.3.3-1.10} [IvToolkit] (minecraft.jar) 
	UCHIJAAAA	rpghud{3.6.6} [RPG-Hud] ([1.10.2] RPG-HUD-3.6.6.jar) 
	UCHIJAAAA	movillages{1.5.2} [Mo' Villages] ([1.9.4]Mo' Villages-1.5.2.jar) 
	UCHIJAAAA	Baubles{1.3.11} [Baubles] (Baubles-1.10.2-1.3.11.jar) 
	UCHIJAAAA	Botania{r1.9-341} [Botania] (Botania r1.9-341.jar) 
	UCHIJAAAA	AkashicTome{1.0-5} [Akashic Tome] (AkashicTome-1.0-5.jar) 
	UCHIJAAAA	animania{1.2.2} [Animania] (animania-1.10.2-1.2.2.jar) 
	UCHIJAAAA	astralsorcery{1.4.5} [Astral Sorcery] (astralsorcery-1.4.5.jar) 
	UCHIJAAAA	Quark{r1.1-70} [Quark] (Quark-r1.1-70.jar) 
	UCHIJAAAA	AutoRegLib{1.0-2} [AutoRegLib] (AutoRegLib-1.0-2.jar) 
	UCHIJAAAA	bedrockores{1.2.0.30} [Bedrock Ores] (Bedrock Ores-MC1.10.2-1.2.0.30.jar) 
	UCHIJAAAA	Chameleon{1.10.2-2.3.0} [Chameleon] (Chameleon-1.10.2-2.3.0.jar) 
	UCHIJAAAA	chisel{MC1.10.2-0.0.13.30} [Chisel] (Chisel-MC1.10.2-0.0.13.30.jar) 
	UCHIJAAAA	ctm{MC1.10.2-0.2.2.24} [CTM] (CTM-MC1.10.2-0.2.2.24.jar) 
	UCHIJAAAA	cyclopscore{0.10.8} [Cyclops Core] (CyclopsCore-1.9.4-0.10.8.jar) 
	UCHIJAAAA	D3Core{1.3.1} [D³Core] (D3Core-1.9.4-1.3.1.jar) 
	UCHIJAAAA	dailies{1.10.2-6} [Bailey's Dailies] (dailies-1.10.2-6.jar) 
	UCHIJAAAA	dldungeonsjdg{1.9.2} [Doomlike Dungeons] (DoomlikeDungeonsJBG-1.9.2-MC1.10.jar) 
	UCHIJAAAA	endercore{1.10.2-0.4.1.66-beta} [EnderCore] (EnderCore-1.10.2-0.4.1.66-beta.jar) 
	UCHIJAAAA	Waila{1.8.17-B31_1.10.2} [Waila] (Hwyla-1.8.17-B31_1.10.2.jar) 
	UCHIJAAAA	JEI{3.14.7.419} [Just Enough Items] (jei_1.10.2-3.14.7.419.jar) 
	UCHIJAAAA	EnderIO{1.10.2-3.1.183} [Ender IO] (EnderIO-1.10.2-3.1.183.jar) 
	UCHIJAAAA	everlastingabilities{1.2.5} [EverlastingAbilities] (EverlastingAbilities-1.9.4-1.2.5.jar) 
	UCHIJAAAA	extrautils2{1.0} [Extra Utilities 2] (extrautils2-1.10.2-1.6.1.jar) 
	UCHIJAAAA	fastleafdecay{v11} [Fast Leaf Decay] (FastLeafDecay-v11.jar) 
	UCHIJAAAA	forgelin{1.5.1} [Shadowfacts' Forgelin] (Forgelin-1.5.1.jar) 
	UCHIJAAAA	ftbl{0.0.0} [FTBLib] (FTBLib-1.1x-3.6.5.jar) 
	UCHIJAAAA	ftbu{0.0.0} [FTBUtilities] (FTBUtilities-1.1x-3.6.5.jar) 
	UCHIJAAAA	lit{@VERSION@} [Little Insignificant Things] (LittleInsignificantThings-1.10.2-3.0.4.jar) 
	UCHIJAAAA	silentlib{2.1.5} [Silent Lib] (SilentLib-1.10.2-2.1.5-63.jar) 
	UCHIJAAAA	silentgems{2.3.16} [Silent's Gems] (SilentsGems-1.11.x-2.3.16-186.jar) 
	UCHIJAAAA	gemulation{1.10.2-4.0.1} [Gemulation] (Gemulation-1.10.2-4.0.1.jar) 
	UCHIJAAAA	immersiveengineering{0.10-61} [Immersive Engineering] (ImmersiveEngineering-0.10-61.jar) 
	UCHIJAAAA	inventorytweaks{1.61-58-a1fd884} [Inventory Tweaks] (InventoryTweaks-1.61-58.jar) 
	UCHIJAAAA	ironbackpacks{1.10.2-2.2.31} [Iron Backpacks] (IronBackpacks-1.10.2-2.2.31.jar) 
	UCHIJAAAA	journeymap{1.10.2-5.4.7} [JourneyMap] (journeymap-1.10.2-5.4.7.jar) 
	UCHIJAAAA	levels{r1.2.8} [Levels] (Levels 2-1.10.2-r1.2.8.jar) 
	UCHIJAAAA	levelup{${version}} [Level Up!] (LevelUp!-0.12.10-1.10.2.jar) 
	UCHIJAAAA	librarianlib{1.10.1} [LibrarianLib] (librarianlib-1.10.1.jar) 
	UCHIJAAAA	minecolonies{@VERSION@} [MineColonies] (minecolonies-universal-1.10.2-0.8.3056.jar) 
	UCHIJAAAA	botanicaladdons{1.10} [Natural Pledge] (NaturalPledge-r1.11.jar) 
	UCHIJAAAA	Neat{1.3-12} [Neat] (Neat 1.3-12.jar) 
	UCHIJAAAA	harvestcraft{1.10.2j} [Pam's HarvestCraft] (Pam's HarvestCraft 1.10.2j.jar) 
	UCHIJAAAA	reccomplex{1.4.5-1.10} [Recurrent Complex] (RecurrentComplex-1.4.5-1.10.jar) 
	UCHIJAAAA	roots{0.208} [Roots] (roots-0.208.jar) 
	UCHIJAAAA	scalinghealth{1.0.8} [Scaling Health] (ScalingHealth-1.11.2-1.0.8-52.jar) 
	UCHIJAAAA	shetiphiancore{3.3.11} [ShetiPhian-Core] (shetiphiancore-1.10.0-3.3.11.jar) 
	UCHIJAAAA	SparksHammers{1.10.2-1.6.5} [Spark's Hammers] (SparksHammers-1.10.2-1.6.5.jar) 
	UCHIJAAAA	StorageDrawers{1.10.2-3.7.10} [Storage Drawers] (StorageDrawers-1.10.2-3.7.10.jar) 
	UCHIJAAAA	terraqueous{1.2.25} [Terraqueous] (terraqueous-1.10.0-1.2.25.jar) 
	UCHIJAAAA	tp{1.2.3} [Tiny Progressions] (tinyprogressions-1.10.2-1.2.3a.jar) 
	UCHIJAAAA	tombmanygraves{1.10.2-2.1.16} [Tomb Many Graves] (TombManyGraves-1.10.2-2.1.16.jar) 
	UCHIJAAAA	toroquest{1.10.2-3.2.3} [ToroQuest] (toroquest-1.10.2-3.2.3.jar) 
	UCHIJAAAA	VillageNames{1.10.2-1.1a} [Village Names] (VillageNames-1.10.2-1.1a.jar) 
	UCHIJAAAA	WailaHarvestability{1.1.10} [Waila Harvestability] (WailaHarvestability-mc1.10.2-1.1.10.jar) 
	UCHIJAAAA	waystones{2.0.13} [Waystones] (Waystones_1.10.2-2.0.13.jar) 
	UCHIJAAAA	librarianliblate{1.10.1} [LibrarianLib Stage 2] (librarianlib-1.10.1.jar) 
	Loaded coremods (and transformers): 
ShetiPhian-ASM (shetiphiancore-1.10.0-3.3.11.jar)
  shetiphian.core.asm.ClassTransformer
ForgelinPlugin (Forgelin-1.5.1.jar)
  
CTMCorePlugin (CTM-MC1.10.2-0.2.2.24.jar)
  team.chisel.ctm.client.asm.CTMTransformer
LoadingPlugin (Quark-r1.1-70.jar)
  vazkii.quark.base.asm.ClassTransformer
SHLoadingPlugin (ScalingHealth-1.11.2-1.0.8-52.jar)
  net.silentchaos512.scalinghealth.asm.SHClassTransformer
EnderCorePlugin (EnderCore-1.10.2-0.4.1.66-beta.jar)
  com.enderio.core.common.transform.EnderCoreTransformer
FMLPlugin (InventoryTweaks-1.61-58.jar)
  invtweaks.forge.asm.ContainerTransformer
IvToolkit (IvToolkit-1.3.3-1.10.jar)
  
AstralCore (astralsorcery-1.4.5.jar)
  
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	D3Core: Debug: false Silliness: true AprilFools: true PastPost:true
	EnderIO: No known problems detected.
	Detailed RF API diagnostics:
                  * RF API class 'EnergyStorage' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/LittleInsignificantThings-1.10.2-3.0.4.jar!/cofh/api/energy/EnergyStorage.class
                  * RF API class 'IEnergyConnection' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/extrautils2-1.10.2-1.6.1.jar!/cofh/api/energy/IEnergyConnection.class
                  * RF API class 'IEnergyContainerItem' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/extrautils2-1.10.2-1.6.1.jar!/cofh/api/energy/IEnergyContainerItem.class
                  * RF API class 'IEnergyHandler' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/extrautils2-1.10.2-1.6.1.jar!/cofh/api/energy/IEnergyHandler.class
                  * RF API class 'IEnergyProvider' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/extrautils2-1.10.2-1.6.1.jar!/cofh/api/energy/IEnergyProvider.class
                  * RF API class 'IEnergyReceiver' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/extrautils2-1.10.2-1.6.1.jar!/cofh/api/energy/IEnergyReceiver.class
                  * RF API class 'IEnergyStorage' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/extrautils2-1.10.2-1.6.1.jar!/cofh/api/energy/IEnergyStorage.class
                  * RF API class 'ItemEnergyContainer' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/LittleInsignificantThings-1.10.2-3.0.4.jar!/cofh/api/energy/ItemEnergyContainer.class
                  * RF API class 'TileEnergyHandler' is loaded from: jar:file:/C:/Users/gabor/OneDrive/Dokumentumok/Curse/Minecraft/Instances/Astral%20Exploration/mods/LittleInsignificantThings-1.10.2-3.0.4.jar!/cofh/api/energy/TileEnergyHandler.class
	Detailed Tesla API diagnostics:
                  * Tesla API class 'Tesla' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.Tesla)
                  * Tesla API class 'TeslaCapabilities' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.capability.TeslaCapabilities)
                  * Tesla API class 'ITeslaConsumer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.ITeslaConsumer)
                  * Tesla API class 'ITeslaHolder' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.ITeslaHolder)
                  * Tesla API class 'ITeslaProducer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.ITeslaProducer)
                  * Tesla API class 'BaseTeslaContainer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.BaseTeslaContainer)
                  * Tesla API class 'BaseTeslaContainerProvider' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.BaseTeslaContainerProvider)
                  * Tesla API class 'InfiniteTeslaConsumer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaConsumer)
                  * Tesla API class 'InfiniteTeslaConsumerProvider' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaConsumerProvider)
                  * Tesla API class 'InfiniteTeslaProducer' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaProducer)
                  * Tesla API class 'InfiniteTeslaProducerProvider' could not be loaded (reason: java.lang.ClassNotFoundException: net.darkhax.tesla.api.implementation.InfiniteTeslaProducerProvider)

	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
	!!!You are looking at the diagnostics information, not at the crash. Scroll up!!!
	!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

	Profiler Position: N/A (disabled)
	Player Count: 1 / 8; [EntityPlayerMP['hron84'/296, l='Naruto', x=87.50, y=40.00, z=32.15]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'
```

**Comment by fnuecke** [comment]
73ca4d92aa7819708ba17a2bd81bc5bd43b5d68c

Should be fixed in v1.2.2!

**Comment by hron84** [comment]
@fnuecke thanks, I'll check it as soon as I can.
